internal/manifests/collector/configmap.go

```diff
@@ -34,3 +38,35 @@ func ConfigMap(params manifests.Params) (*corev1.ConfigMap, error) {
 		},
 	}, nil
 }
+
+func PrometheusConfigMap(params manifests.Params) (*corev1.ConfigMap, error) {
+	name := naming.PrometheusConfigMap(params.OtelCol.Name)
+	labels := manifestutils.Labels(params.OtelCol.ObjectMeta, name, params.OtelCol.Spec.Image, ComponentAmazonCloudWatchAgent, []string{})
```

Comment:
Does this label look correct?

Reply:
<img width="542" alt="Screenshot 2024-09-10 at 12 39 05 PM" src="https://github.com/user-attachments/assets/45e94475-993b-4024-b47d-981a0c43a166">

Reply:
Remove the version label

Reply:
We will have to do the same for the `target-allocator`, but instead of removing it, use the `target-allocator` version.